src/index.js

```diff
@@ -1,8 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
```

Comment:
why did you remove this?

Reply:
I removed it because MUI is basically like inline css and I didnt think i would need a index.css file.